var/spack/repos/builtin/packages/ramulator/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Ramulator(MakefilePackage):
+    """
+    Ramulator is a fast and cycle-accurate DRAM simulator that supports
+    a wide array of commercial, as well as academic, DRAM standards.
+    """
+
+    homepage = "https://github.com/CMU-SAFARI/ramulator"
+    git = "https://github.com/CMU-SAFARI/ramulator"
+
+    version('sst', git="git@github.com:CMU-SAFARI/ramulator.git",
```

Comment:
`git` is not needed here since you define it above

Reply:
Touché